spring-ws-core/src/test/java/org/springframework/ws/soap/addressing/AbstractWsAddressingTestCase.java

```diff
@@ -59,12 +59,6 @@ protected SaajSoapMessage loadSaajMessage(String fileName) throws SOAPException,
 	protected void assertXMLEqual(String message, SaajSoapMessage expected, SaajSoapMessage result) {
 		Document expectedDocument = expected.getSaajMessage().getSOAPPart();
 		Document resultDocument = result.getSaajMessage().getSOAPPart();
-		org.custommonkey.xmlunit.XMLAssert.assertXMLEqual(message, expectedDocument, resultDocument);
-	}
-
-	protected void assertXMLSimilar(String message, SaajSoapMessage expected, SaajSoapMessage result) {
-		Document expectedDocument = expected.getSaajMessage().getSOAPPart();
-		Document resultDocument = result.getSaajMessage().getSOAPPart();
-		org.custommonkey.xmlunit.XMLAssert.assertXMLEqual(compareXML(expectedDocument, resultDocument), false);
```

Comment:
This is wrong.
`XMLAssert.assertXMLEqual(compareXML(expectedDocument, resultDocument), false)` returns the opposite of what is needed.